README.md

```diff
@@ -2,6 +2,15 @@
 
 Discussion agents.
 
+# Windows installation 
+```
+1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console
+2.conda create -n my_env python=3.10.13
```

Comment:
Code can go in the code block and the text can go outside of the code block. Maybe add a statement above the code block to describe the step, the command, and the flags. 

What is `conda create`?
What is `-n`?
What is the `my_env` argument?
What is the `python=3.10.13`?